scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
```

Comment:
TARS:<br/>:stop_sign: **Security Vulnerability**: Hardcoded API Key

Storing API keys directly in the source code is a severe security risk[^1]. This exposes sensitive information and can lead to unauthorized access if the code is compromised.

Recommendation: Use environment variables or a secure secret management system to store and retrieve API keys.
```suggestion
const apiKey = process.env.API_KEY
if (!apiKey) {
    throw new Error('API_KEY environment variable is not set')
}
```